drakrun/drakrun/main.py

```diff
@@ -275,6 +275,14 @@ def _get_start_command(self, extension, sample, file_path):
             start_command = None
         return start_command
 
+    def _karton_safe_get_headers(self, task, key, fallback):
+        ret = task.headers.get(key, fallback)
+        if ret is None:
```

Comment:
I think this is going to enter this `if` only if `fallback = None`, but we don't have any case like this here. So this would be a dead code in this form.

Reply:
I had talked to @chivay actually regarding this, He had told `karton` had some bug in past which did not consider the `fallback` ( which is mentioned in the issue ). So, we should emit a warning in case this happens. That's what I have done in the PR

Reply:
Yes, it happened and it probably may happen again that headers got a explicit `extension=null`. It's not very likely but it would be better to issue a warning and fall back to a default than to crash.